src/components/ClaimFilter.js

```diff
@@ -255,7 +255,7 @@ const mapStateToProps = (state) => ({
 });
 
 const mapDispatchToProps = (dispatch) => {
-  return bindActionCreators({ selectClaimAdmin, selectHealthFacility, selectDistrict, selectRegion }, dispatch);
+  return bindActionCreators({ selectClaimAdmin, selectHealthFacility, selectDistrict, selectRegion, clearClaim }, dispatch);
```

Comment:
This record might be also a little bit too long. Same case as in previous comment, what's the purpose of using here _clearClaim_ action?